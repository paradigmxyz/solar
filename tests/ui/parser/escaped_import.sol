import "\?"; //~ ERROR unknown character escape
