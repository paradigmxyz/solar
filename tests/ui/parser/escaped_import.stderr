error: unknown character escape
  --> $DIR/escaped_import.sol:1:10
   |
LL | import "\?";
   |          ^
   |

error: aborting due to 1 previous error

